File: Week 4 Day 4 : Understanding C# Iterative Statements/2. Reversing a number.cs
using System;

class Program {
    public static void Main(String[] args) {
        Console.WriteLine("Enter the number :");
        int n=Convert.ToInt32(Console.ReadLine());
        int rev=0;
        while(n>0)
        {
            int d=n%10;
            rev=rev*10+d;
            n=n/10;
        }
        Console.WriteLine("Reverse of the number is "+rev);
    }
}

File: Week1 Day3: Looping Statement - While and DoWhile Loop/2. Reversing a number.java
import java.util.*;
class Main {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the number:");
        int n=sc.nextInt();
        int rev=0;
        while(n>0)
        {
            int d=n%10;
            rev=rev*10+d;
            n=n/10;
            
        }
        System.out.println("Reverse of the number is "+rev);
            // Fill your code here
        
     }
}
